### PR TITLE
DOC: Review keras.losses 

### PR DESCRIPTION
**Pull request by lc0**
- fix failing example for `tf.keras.losses.Poisson`
- update numbers for `tf.keras.losses.Poisson`
- update numbers for `tf.keras.losses.KLDivergence`
- add example for `tf.keras.losses.kullback_leibler_divergence`

btw, @lamberta do we have any way to reduce duplication of such docs? 
currently we support support for losses as a function as well as object-style

Pushed during Munich TFDocSprint cc @dynamicwebpaige

### CODE REVIEW

**Comment by lamberta** [comment]
@lc0 Thank you, Sergii.
Reduce duplication from the non-TF Keras API reference? Unfortunately, no. The tf.keras docs were forked and there is no automatic way to keep them in sync. But there's now a [Keras SIG](https://github.com/keras-team/governance) and that may be worth bringing up. cc @fchollet

**Comment by lc0** [comment]
any updates on this cc @pavithrasv?

**Review by dynamicwebpaige — APPROVED** [review]
Thanks for the updates! 😊

**Comment by lc0** [comment]
Seems like the failing test is not related to my fixing a doc

**Review by pavithrasv — APPROVED** [review]

**Comment by lc0** [comment]
another ping on this one. The PR has a label `ready to pull` but not merged yet for quite some time already. Might be because of failing tests, but as I said above, these have nothing to do with my docs changes.

Would anybody take a look from your side: what is still missing?

**Review by pavithrasv — APPROVED** [review]

**Comment by lc0** [comment]
Thank you everyone involved!
